src/routes/jira/jira-router.ts

```diff
@@ -18,3 +19,4 @@ JiraRouter.use("/events", JiraEventsRouter);
 JiraRouter.use("/select-version", JiraSelectVersionRouter);
 JiraRouter.use("/server-url", JiraServerUrlRouter);
 JiraRouter.use("/app-creation", JiraAppCreationRouter);
+JiraRouter.use("/ghe-servers", JiraGheServersRouter);
```

Comment:
we don't have a create a new router for every new path that only has a `GET` call on it.  We can just do `JiraRouter.get("/ghe-servers", csrfMiddleware, JiraJwtTokenMiddleware, JiraGheServers)` here

Reply:
Updated to this and removed the router file.